app/models/capital.rb

```diff
@@ -71,9 +71,10 @@ def partner_has_premium_bonds
     super if requires_full_capital?
   end
 
-  def usual_property_details_required?
+  def usual_property_details_required? # rubocop:disable Metrics/AbcSize
     return false unless FeatureFlags.property_ownership_validation.enabled?
     return false unless requires_full_capital?
+    return false if crime_application.applicant.residence_type.blank?
```

Comment:
Why was this added? 🤔

Reply:
There was an instance where a user skipped ahead and go to this question before answering the `residence_type` question, and the page threw an error because of this. This doesn't fix the potentially bigger issue of users being able to jump ahead like that, but thought I'd add this to prevent it from happening again. Here's more about what happened: https://mojdt.slack.com/archives/C056U956ESH/p1738932998077209